CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# What do I need to know to help?
+
+When contributing to this repository, please feel free to discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+
+## How do I make a contribution?
+
+If you've never made an open source contribution before or you are wondering how contributions work in our project? Here's a quick rundown!
+
+1. Fork the [repository](https://github.com/aaryan2134/JPMC-Quant-Challenge-2022) to your account. This means that you will have a copy of the repository under `<your-GitHub-username>/<repository-name>`.
+2. Clone the repository to your local machine using by copying the link provided under `HTTPS` when you click on the green button labeled `code` on the repository page
+
+    ```sh
+    $ git clone https://github.com/<your-GitHub-username>/Url-Shortener-API.git
+    ```
+
+4. Create a new branch for your changes or fix 
+
+    ```sh
+     $ git checkout -b <branch-name>
+    ```
+
+5. Setup the project in your local by following the steps listed in the [README.md](https://github.com/aaryan2134/JPMC-Quant-Challenge-2022/blob/main/README.md) file.
+6. Open the project in a code editor and begin working on it.
+7. Add the contents of the changed files to the "snapshot" git uses to manage the state of the project, also known as the index
+
+    ```sh
+    $ git add .
+    ```
+
+8. Add a descriptive commit message
+
+    ```sh
+    $ git commit -m "Insert a short message of the changes made here"
+    ```
+
+9. Push the changes to the remote repository
+
+    ```sh
+    $ git push -u origin <branch-name>
+    ```
+
+10. Submit a pull request to the upstream repository.
+11. Title the pull request with a short description of the changes you made and **request a review from a maintainer/reviewer**. For example, you can title an issue like so "Added John Doe to the project". In the description of the pull request, be sure to explain the changes that you made and any questions you have for the maintainer. It's OK if your pull request is not perfect (no pull request is), the reviewer will be able to help you fix any problems and improve it!
```

Comment:
Please change the example to something which is more appropriate to the repository.